platforms/shared/configuration/environment-setup.yaml

```diff
@@ -59,7 +59,7 @@
         kubeconfig_path: "{{ item.k8s.config_file }}"
         kubecontext: "{{ item.k8s.context }}"
       with_items: "{{ network.organizations }}"
-      when: "item.cloud_provider == 'aws'"
+      when: "item.cloud_provider == 'aws' and item.k8s.provider == 'EKS'"
```

Comment:
K8s.provider is removed now. I think you should use a different cloud-provider for bare metal because essentially it will be cloud-provider independent

Reply:
That means that cloud_provider=aws will always assume that K8S is via EKS? For Indy we still have to know that we're in AWS because we stick to the Ambassador provisioning a NLB using AWS EIPs.

Reply:
Yes, for that maybe use a different value, like aws-something

Reply:
Ok, so aws-baremetal it is...